contracts/cw4-stake/src/contract.rs

```diff
@@ -36,10 +36,10 @@ pub fn instantiate(
     ADMIN.set(deps.branch(), maybe_addr(api, msg.admin)?)?;
 
     // min_bond is at least 1, so 0 stake -> non-membership
-    let min_bond = match msg.min_bond {
-        Uint128(0) => Uint128(1),
+    let min_bond = Uint128::new(match msg.min_bond.u128() {
+        0 => 1,
         v => v,
-    };
+    });
```

Comment:
Can't we use `let min_bond = std::cmp::max(msg.min_bond, Uint128::new(1));` instead?